app/assets/javascripts/admin/addon/components/webhook-events.js

```diff
@@ -78,6 +93,20 @@ export default class WebhookEvents extends Component {
       this.pingEnabled = true;
     }
 
+    if (data.type === "redelivered") {
+      const event = this.events.find((e) => {
```

Comment:
Minor but you can do this instead:

```suggestion
      const event = this.events.findBy("id", data.web_hook_event.id)
```